Source/WebCore/platform/graphics/ca/PlatformCAFilters.h

```diff
@@ -31,14 +31,21 @@
 #include <wtf/RetainPtr.h>
 
 OBJC_CLASS NSValue;
+OBJC_CLASS CAPresentationModifier;
+OBJC_CLASS CAPresentationModifierGroup;
 
 namespace WebCore {
 
 class PlatformCALayer;
 
+using TypedFilterPresentationModifier = std::pair<FilterOperation::Type, RetainPtr<CAPresentationModifier>>;
+
 class PlatformCAFilters {
 public:
     WEBCORE_EXPORT static void setFiltersOnLayer(PlatformLayer*, const FilterOperations&);
+    WEBCORE_EXPORT static void presentationModifiersForFilters(const FilterOperations& initialFilters, const FilterOperations* canonicalFilters, Vector<TypedFilterPresentationModifier>& presentationModifiers, RetainPtr<CAPresentationModifierGroup>&);
+    WEBCORE_EXPORT static void updatePresentationModifiersForFilters(const FilterOperations& filters, const Vector<TypedFilterPresentationModifier>& presentationModifiers);
```

Comment:
You can remove `filters` here.

Reply:
Good point, this whole set of helpers are about filters!